src/roadmap/releases.md

```diff
@@ -193,15 +205,23 @@ Change the `generate_params.toml` file in **each** new chart as follows:
   1. If there's a `[workflowManager]` section, change `versionsApiURL` to
      `"https://versions.deis.com"` and `doctorApiURL` to `"https://doctor.deis.com"`
 
-Change the `workflow-$WORKFLOW_RELEASE/tpl/deis-controller-deployment.yaml` file:
+Change the `workflow-$WORKFLOW_RELEASE/tpl/deis-controller-deployment.yaml` and
+`workflow-$WORKFLOW_RELEASE/tpl/deis-controller-rc.yaml` files:
 
   1. Remove the `KUBERNETES_POD_TERMINATION_GRACE_PERIOD_SECONDS` env var
 
+Change any "-dev" labels to reference the new release instead, and remove test wording:
```

Comment:
I'm cool with this but lets try to automate it sooner than later, it's simple `sed`